__tests__/feature/userAuthentication.test.js

```diff
@@ -71,12 +71,15 @@ describe('user authentication', () => {
 
     awsCognito.CognitoUser.mockImplementation(MockCognitoUserAuthenticationFailure)
 
+    // we don't expect any alerts to be present yet...
+    expect(screen.queryByRole('alert')).not.toBeInTheDocument()
+
     // try to login
     fireEvent.change(screen.getByLabelText('User name'), { target: { value: 'baz.le.quux@example.edu' } })
     fireEvent.change(screen.getByLabelText('Password'), { target: { value: 'password1' } })
     fireEvent.click(screen.getByRole('button', { name: 'Login' }))
 
-    // an error message should be presented
-    await screen.findByText(/Incorrect username or password./)
+    // ...and an accessible alert should be present now that a failed login attempt was made
+    await screen.findByRole('alert', { name: 'Incorrect username or password.' })
```

Comment:
fails prior to the second commit in this PR:
![Screen Shot 2020-08-28 at 2 42 33 PM](https://user-images.githubusercontent.com/7741604/91617442-eeaf5900-e93c-11ea-9a7d-1ae1a0813875.png)